packages/react/src/themes/teams/staticStyles/globalStyles.ts

```diff
@@ -1,9 +1,6 @@
 import { StaticStyleFunction } from '../../types'
 
 const globalStyles: StaticStyleFunction = siteVars => ({
-  html: {
-    fontSize: siteVars.htmlFontSize,
```

Comment:
the result of this line is that for consumer now `1rem = htmlFontSize = 16px`, while previously it was `1rem = htmlFontSize = 10px`. Are we sure this doesn't provide any implications for the client (i.e. another set of static styles is used on the client side that ensure that `htmlFontSize` remains to be `10px`)?

Reply:
talked through and have verified client scenario - there shouldn't be any related regressions there 👍